### ISSUE TITLE
6.0 - custom user-agent lost on reloading

### ISSUE THREAD

**Issue by xverges** [nwjs/nw.js#840]
A custom user-agent string set in `package.json` is lost when the reload button is used.

Probably related to #597 and #806

**Comment by jharia** [comment]
Bump - has anyone here figured out a way around this?

**Comment by jgoalby** [comment]
Have you tried capturing the value before the reload and then setting it after the reload?

**Comment by vivvvi** [comment]
I've also hit this hurdle wrapping my chrome web-app in 'the node'.
I heavily rely on appcache manifests to cache the app and images to disk, and a quirk of that is required browser refreshes.

I have custom UA set in node's package.json that the server can identify the client and dish some additional 'node code' which works as expected first time, however the refresh then renders the $_SERVER['HTTP_USER_AGENT'] server side vars in PHP return empty thereafter.

I hide the toolbar in my fullscreen web-app however debugging I noticed if I added a ? to the address, the correct USER_AGENT returned.

Unfortunately without LOTS of recoding of my webapp, or using node's 'special iframes', for me this is a deal breaker. :(
